config/authentication.go

```diff
@@ -14,7 +14,7 @@ func AuthenticationHandler(next http.Handler) http.Handler {
 		if strings.Contains(r.Header.Get("Authorization"), "Bearer") {
 			err := ValidateToken(strings.TrimPrefix(r.Header.Get("Authorization"), "Bearer "))
 			if err != nil {
-				log.Warning("Error token %+v", err)
+				log.Warning("Token error: ", err)
```

Comment:
It took me a moment to accept that this was OK, I originally thought the fix should be:

`  log.Warningf("Error token %+v", err)`

But your way should work as well.